### DISCUSSION TITLE
Add a Score column showing QSO Points and Multiplier value

### DISCUSSION THREAD

**Discussion by w7sst** [w7sst/MorseRunner#171] [Ideas]
Many contests are showing the "Pref" column to render the prefix of the current callsign. While this might be useful for the original WPX contest, it is not useful for many of the other contests. While working on the new IARU HF Contest, I got the idea of using this column to show the Points and optional Multiplier values for the current QSO.

![image](https://user-images.githubusercontent.com/106113737/207925814-cfc34551-72a0-420d-bada-55914a18a237.png)

The Score column `Pts(M)` shows the verified Points and optional Multiplier value. These values add up to match the Verified Score summary section in the lower-right of the screen.

I'd like your feedback on this idea. If we like this, we can convert many of the contests to add this Score Column to the screen report.

An alternate title for this column is `Score`.

Download [this build](https://nextcloud.brashlerphoto.com/s/jJ4LJtFWtWTZsSG) to give this feature a try.

Thoughts? Feedback?

Thank you.

73, Mike W7SST

@f6fvy @k6ok @tekenny @n2ic @scotthibbs @DF9TS @jr8ppg

**Comment by scotthibbs** [comment]
Seems I'm missing this file...
[image: image.png]

but I can already tell you that I love the idea of the points column!
73
KD4SIR
Scott Hibbs
812-581-0631
https://smile.amazon.com/ch/35-6042852

**Reply by w7sst** [reply]
I edited the post and re-pasted the image. Try it again now. @scotthibbs

**Comment by K6OK** [comment]
Excellent.  A dynamic (by contest) points and mults column is two thumbs up 👍👍
73 Jim K6OK

**Comment by scotthibbs** [comment]
I get the same error. I used the previous download link, was there a new
link?

73
KD4SIR
Scott Hibbs
812-581-0631
https://smile.amazon.com/ch/35-6042852

**Reply by w7sst** [reply]
Are you clicking the link from within github.com, https://github.com/w7sst/MorseRunner/discussions/171? Or from within the email?

So you are seeing the error when you view this page?  https://github.com/w7sst/MorseRunner/discussions/171

Interesting.

BTW, I am heading out of town tomorrow for 10 days and will not have computer access. I will be able to check mail with my phone, so will try to respond as I can. Merry Christmas!

**Comment by scotthibbs** [comment]
Within the email or #171. I click the first message where it says
 "Download [this build](https://nextcloud.brashlerphoto.com/s/jJ4LJtFWtWTZsSG) to give this feature a try."
I'm not getting the IARU_HF.txt file in the folder. 

Not a biggie. I've been doing the Field Day contest.. Get on your trip and have a great time! 

Merry Christmas Everyone!
Scott

**Reply by w7sst** [reply]
Sorry about that. I misunderstood your original post about the bad link. I thought you couldn't see the graphic posted above. But instead you were not getting the IARU_HF.txt file. Good find. I added the missing file and [regenerated the zip file](https://nextcloud.brashlerphoto.com/s/jJ4LJtFWtWTZsSG). This should work for you now.

**Comment by f6fvy** [comment]
Apparently, it doesn't work if the callsign you're using is an HQ (ex TM0HQ / Exch. REF). We get an empty error message and there is no caller.

**Comment by w7sst** [comment]
Thank you for finding this. My guess is the call sign is not recognized by
our limited DXCC file. I will also need to improve error reporting for this
case.

We are leaving town today for a few weeks. I’ll look at this after I get
back.

73, Mike

**Reply by f6fvy** [reply]
> My guess is the call sign is not recognized by our limited DXCC file

Indeed. Altering line 125 of ARRL.LIST fixes it. But I remain convinced that this file is not suitable for this use in the future.

**Comment by w7sst** [comment]
I completely agree with you.

Laurent, do you have a .cty file reader (any language) that I or someone
can port to pascal? If we do it correctly, it could be packaged as a
reusable component (unit) for others to use.

Thanks,
Mike
